src/drivers/mb12xx/mb12xx.cpp

```diff
@@ -259,6 +277,38 @@ MB12XX::init()
 			debug("failed to create sensor_range_finder object. Did you start uOrb?");
 		}
 	}
+	usleep(200000);
+
+	/* check for connected rangefinders on each i2c port:
+	   We start from i2c base address (0x70 = 112) and count downwards
+	   So second iteration it uses i2c address 111, third iteration 110 and so on*/
+	for(int counter = 0; counter <= MB12XX_MAX_RANGEFINDERS; counter++) {
+		_index_counter = MB12XX_BASEADDR-counter;	/* set temp sonar i2c address to base adress - counter */
+		set_address(_index_counter);			/* set I2c port to temp sonar i2c adress */
+		int ret2 = measure();
+		
+		if(ret2 == 0) { /* sonar is present -> store address_index in array */
+			addr_ind.push_back(_index_counter);
+			log("sonar added");
+			_latest_sonar_measurements.push_back(200);
+		}
+		usleep(200000);
+	}
+	_index_counter = MB12XX_BASEADDR;	
+	set_address(_index_counter); /* set i2c port back to base adress for rest of driver */
+
+	/* if only one sonar detected, no special timing is required between firing, so use default */	
+	if (addr_ind.size() == 1){
+	_cycling_rate = MB12XX_CONVERSION_INTERVAL;
+	}
+	else{
+	_cycling_rate = TICKS_BETWEEN_SUCCESIVE_FIRES;
+	}
+	/* show the connected sonars in terminal */
+	for(int i = 0; i < addr_ind.size(); i++) {
+		log("sonar %d met address %d toegevoegd",(i+1),addr_ind[i]);
```

Comment:
My dutch is a bit rusty - maybe change to english? Since this is a one-time enumeration during boot the print seems to be ok.